Sources/CollectionsBenchmark/Basics/Blackhole.swift

```diff
@@ -15,6 +15,6 @@
 /// get used, and this could potentially interfere with the accuracy of a
 /// benchmark. To defeat these optimizations, pass such unused results to
 /// this function so that the compiler considers them used.
-@inline(never)
+@_optimize(none)
```

Comment:
I think I'd prefer to keep the inlining attribute in there, if only as a belt-and-suspenders thing.

```suggestion
@inline(never)
@_optimize(none)
```